### PR TITLE
Added Transport OpenRPC Document for the MultiChain API

### PR DESCRIPTION
**Pull request by shanejonas**
image

### CODE REVIEW

File: transport/openrpc.yaml
Comment (jiexi) [inline transport/openrpc.yaml:1]: is it more accurate to differentiate this as a transport or as multichain specific?

**Review by jiexi — DISMISSED** [review]
looks great

File: transport/openrpc.yaml
Comment (adonesky1) [reply transport/openrpc.yaml:1]: yeah I feel like it makes more sense to name this folder `multichain`?

File: transport/openrpc.yaml [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        methods:
+          description: "Methods that the wallet must support in order to be used with this provider."
+          type: "array"
+          items:
+            type: "string"
+        notifications:
+          description: "notifications that the wallet must support in order to be used with this provider."
+          type: "array"
+          items:
+            type: "string"
```
Comment (adonesky1) [inline transport/openrpc.yaml:168]: according to [CAIP-217](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-217.md), `methods` and `notifications` are both required properties

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Pull request merged**